libs/components/src/next/properties/stories/vl-properties-stories-doc.mdx

```diff
@@ -0,0 +1,108 @@
+import {ArgTypes, Canvas} from '@storybook/addon-docs';
+
+import * as VlPropertiesStories from './vl-properties.stories';
+
+# Properties - next
+
+Gebruik `properties-next` om properties te tonen op een pagina. Je kan de properties specifiëren als inner-html of
```

Comment:
Gebruik de `properties-next` component om...

Reply:
Eigenlijk was dat om de discussie is het nu: 'de component' of 'het component' te omzeilen